src/stack.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+export default class Stack {
+  constructor(initialValues=[]) {
+    this.elements = initialValues
+  }
+
+  push(value) {
+    this.elements[this.elements.length] = value
+  }
+
+  pop() {
+    if(this.elements.length === 0){
+      return null
+    }
+    let result = this.elements.slice(-1)
+    this.elements.splice(-1)
+    return result
+
+  }
+
+  peek() {
+    if (this.elements.length === 0){
+      return null
+    }
+    return this.elements[this.elements.length -1]
+  }
+
+  isEmpty() {
+    if (this.elements.length === 0 ){
+      return true
```

Comment:
you never need to do this `if (X) { return true } else { return false }`. Just do that:

```js
isEmpty() {
  return this.elements.length === 0
}
```